docker/nginx/nginx.conf

```diff
@@ -0,0 +1,88 @@
+#ssl_certificate /etc/letsencrypt/live/myopenhab.org/fullchain.pem;
+#ssl_certificate_key /etc/letsencrypt/live/myopenhab.org/privkey.pem;
+worker_processes 4;
+
+events { worker_connections 1024; }
+
+http {
+
+upstream app_server {
+        server app-1:3000 fail_timeout=5s max_fails=5;
+}
+
+server {
+    listen 8080 default_server;
```

Comment:
Yep is now set to port 80. Still some css struggle which nginx doesn't recognize.
You can find the new version here https://github.com/openhab/openhab-cloud/pull/91/commits/27b17887800c1cfbbebeba99f0dad4654cdd160c